src/test/javascript/spec/component/admin/system-notification-management/system-notification-management.component.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('SystemNotificationManagementComponent', () => {
                 MockPipe(ArtemisTranslatePipe),
                 MockRouterLinkDirective,
                 MockDirective(RouterOutlet),
-                MockDirective(AlertComponent),
+                MockDirective(AlertOverlayComponent),
```

Comment:
```suggestion
                MockComponent(AlertOverlayComponent),
```
Using directive here was probably wrong before as well.